app/metrics.yaml

```diff
@@ -254,3 +260,249 @@ control:
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  triple_window_opened_time:
+    type: timing_distribution
+    description: >
+      Measuring how long a user uses triple windows in a session.
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  single_pri_window_opened_time:
+    type: timing_distribution
+    description: >
+      Measuring how long a user uses single private window in a session.
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  double_pri_window_opened_time:
+    type: timing_distribution
+    description: >
+      Measuring how long a user uses double private windows in a session.
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  triple_pri_window_opened_time:
+    type: timing_distribution
+    description: >
+      Measuring how long a user uses triple private windows in a session.
+    send_in_pings:
+      - session-end
+    time_unit: microsecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  opened_window_count:
+    type: labeled_counter
+    description: >
+      Counting which multiple window mode (single, double, triple)
+      users are using in a session.
+    send_in_pings:
+      - session-end
+    labels:
+      - single
+      - double
+      - triple
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-11-01"
+  opened_pri_window_count:
```

Comment:
I'm not sure we can/should measure this. It's worth specifically checking with data-steward about this.